tests/test_deployments/test_app.py

```diff
@@ -182,7 +178,7 @@ def remove_prefix(text: str, prefix: str) -> str:
             Path(self.tm_home, "data", "priv_validator_state.json"),
         ]
 
-        assert any([f.exists() for f in expected_file_names]), expected_file_names
+        assert all(f.exists() for f in expected_file_names), expected_file_names
```

Comment:
fix

Reply:
I think this had an issue somewhere? Would be good to link to it and resolve it.

Reply:
you are right, #1011